ivy/functional/frontends/numpy/statistics/histograms.py

```diff
@@ -0,0 +1,35 @@
+import ivy
+from ivy.functional.frontends.numpy.func_wrapper import (
+    to_ivy_arrays_and_back,
+    handle_numpy_dtype,
+    from_zero_dim_arrays_to_scalar,
+    handle_numpy_out,
+)
+
+
+@handle_numpy_out
+@handle_numpy_dtype
+@to_ivy_arrays_and_back
+@from_zero_dim_arrays_to_scalar
+def histogram(
+    a,
+    bins=10,
+    range=None,
+    normed=None,
+    weights=None,
+    density=None,
+    *,
+    out=None,
```

Comment:
not sure if I overlooked anything, but I don't think the numpy histogram has out and dtype argument?
ref: https://numpy.org/doc/stable/reference/generated/numpy.histogram.html